app/controllers/pagerouters/EligibilityRouter.scala

```diff
@@ -66,8 +66,10 @@ object EligibilityRouter {
         case Some(DmSpecialOfficeProcessingRequiredCDCS) =>
           if (taxRegime === Sa) whichGenericIneligiblePage(taxRegime)
           else throw new NotImplementedError(s"dmSpecialOfficeProcessingRequiredCDCS ineligibility reason not relevant to ${taxRegime.entryName}")
-        case Some(IsAnMtdCustomer)                       => whichGenericIneligiblePage(taxRegime)
-        case Some(DmSpecialOfficeProcessingRequiredCESA) => whichGenericIneligiblePage(taxRegime)
+        case Some(IsAnMtdCustomer) => whichGenericIneligiblePage(taxRegime)
+        case Some(DmSpecialOfficeProcessingRequiredCESA) =>
+          if (taxRegime === Sia) throw new NotImplementedError(s"DmSpecialOfficeProcessingRequiredCESA ineligibility reason not relevant to ${taxRegime.entryName}")
```

Comment:
do we know if this is the actual behaviour we want? May be worth sticking on the things to check on the other ticket if we're not sure

Reply:
it's not on the list of reasons on the ticket

Reply:
so I would have thought neither this or DmSpecialOfficeProcessingRequiredCDCS should show up for it (though as far as I can tell, DmSpecialOfficeProcessingRequiredCDCS shouldnt show up anyway but has done, for some reason?)

Reply:
ah ok -  you should be able to control what reasons show up on the test-only start page. I think not all tax regimes have the same list so you could use that same mechanismt to control what's available for SIA